.storybook/helpers/useArgTypes.js

```diff
@@ -159,10 +159,13 @@ export function useArgTypes(component, options = { variables: {}}) {
     return [];
   };
   const modifiers = (() => {
-    const options = getModifiers([...cssRules]);
+    const modifiers = __docgenInfo.modifiers || [];
```

Comment:
UiSimpleQuestion has a modifiers in JS XD so we should support extend modifiers object